core/src/main/scala/akka/kafka/ConsumerSettings.scala

```diff
@@ -4,16 +4,35 @@
  */
 package akka.kafka
 
-import scala.concurrent.duration._
-import scala.collection.JavaConverters._
+import java.util.concurrent.TimeUnit
+
 import akka.actor.ActorSystem
 import akka.kafka.internal.ConfigSettings
 import com.typesafe.config.Config
-import org.apache.kafka.clients.consumer.ConsumerConfig
-import org.apache.kafka.clients.consumer.KafkaConsumer
+import org.apache.kafka.clients.consumer.{ConsumerConfig, KafkaConsumer}
 import org.apache.kafka.common.TopicPartition
 import org.apache.kafka.common.serialization.Deserializer
-import java.util.concurrent.TimeUnit
+
+import scala.concurrent.duration._
+
+sealed trait Subscription
```

Comment:
Here it is an API for subscription. We need separate API because we want to have reusable consumer and have to separate consumer configuration and subscription